Support plugin exclusion mechanism (#39)
apache/skywalking-go: docs/en/advanced-features/plugin-exclusion.md

# Plugin Exclusion

> The plugin exclusion is used during the **compilation phase** to prevent certain plugins, as specified by their names, 

Reviewer: ```suggestion
The plugin exclusion is used during the **compilation phase** to exclude specific plugins, through their names, 
```

> The plugin exclusion is used during the **compilation phase** to prevent certain plugins, as specified by their names, 
> from being included in the compilation. Consequently, these excluded plugins will not generate corresponding data when the program is running, such as **Tracing**.

Reviewer: ```suggestion
The plugin exclusion is used during the **compilation phase** to exclude specific plugins, through their names.
Consequently, the codes of these excluded plugins will not be weaved in, then, no relative tracing and metrics.
```

# Plugin Exclusion

The plugin exclusion is used during the **compilation phase** to exclude specific plugins, through their names.
Consequently, the codes of these excluded plugins will not be weaved in, then, no relative tracing and metrics.

## Configuration

```yaml
plugin:
  # List the names of excluded plugins, multiple plugin names should be splitted by ","
  # NOTE: This parameter only takes effect during the compilation phase.

>   excluded: ${SW_AGENT_PLUGIN_EXCLUDES:}

Reviewer: @liuhaoyang I added this, and update the config name(excludes -> excluded), as well as comments. Please update code accordingly.

Author: fixed.
